src/routes/dashboard/admin/events/+page.svelte

```diff
@@ -195,6 +206,11 @@
                        labelText="Base URL:"
                        id="dashboard-admin-input-base-url"
                        bind:value={baseUrl}></Input>
+                <Input ariaLabel="Tage hier die Anzahl der Spare Links ein"
+                       labelText="Anzahl Spare Links:"
```

Comment:
The dashes are not that important, but there's this typo in "Tage".
```suggestion
                <Input ariaLabel="Trage hier die Anzahl der Spare-Links ein"
                       labelText="Anzahl Spare-Links:"
```